libr/anal/sign.c

```diff
@@ -1252,6 +1316,47 @@ static void listVars(RAnal *a, RSignItem *it, int format) {
 	}
 }
 
+static void listTypes(RAnal *a, RSignItem *it, int format) {
+	RListIter *iter = NULL;
+	char *type = NULL;
+	int i = 0;
+
+	if (format == '*') {
```

Comment:
will be good to use pj instead of handwritten json

Reply:
Can you give an example of something that does it this way?

If we do print json with some API utility, we might want to do the same with the rest of the zignature fields, because they're all handwritten.

Reply:
See dadb421e481ba09362edc8171140a2a23fde2abc as example

Reply:
Good. I will open another PR for this refactoring later on, so I can do this for each field and specific cases of zignatures printing.